### PR TITLE
Feature/404

### PR DESCRIPTION
**Pull request by kailasnadh790**
Issue: https://github.com/hlxsites/famous-smoke-cigaradvisor/issues/17

Test URLs:
- https://feature-404--famous-smoke-cigaradvisor--hlxsites.hlx.page/cigaradvisor/fragments/404

### CODE REVIEW

File: cigaradvisor/blocks/hero/hero.css [hunk 1]
```diff
@@ -15,7 +14,7 @@ main .hero.block {
 }
 
 main .hero.block.under-nav {
-  margin-top: -43px;
+  padding-top: 60px;
```
Comment (bstopp) [inline cigaradvisor/blocks/hero/hero.css:17]: You can't change the padding of the hero like this. It impacts other places.

File: cigaradvisor/blocks/hero/hero.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ function buildSearch() {
 }
 
 export default async function decorate(block) {
+  block.classList.add('under-nav');
```
Comment (bstopp) [inline cigaradvisor/blocks/hero/hero.js:17]: This option is an Author decision, it's not something that can be assumed for every hero.

**Review by bstopp — CHANGES_REQUESTED** [review]

File: cigaradvisor/blocks/hero/hero.css (hunk 1 above)
Comment (kailasnadh790) [reply cigaradvisor/blocks/hero/hero.css:17]: Currently this hero is used in home page and pages like 404, I have verified this change works both these places.

File: cigaradvisor/blocks/hero/hero.js (hunk 2 above)
Comment (kailasnadh790) [reply cigaradvisor/blocks/hero/hero.js:17]: I think the hero variation that doesn't go under the nav is on article pages. We are handling that separately as part of article header.

Comment (bstopp) [reply cigaradvisor/blocks/hero/hero.js:17]: Maybe, but you can't assume that every hero is under the nav.

File: cigaradvisor/blocks/hero/hero.js [hunk 3]
```diff
@@ -25,6 +26,10 @@ export default async function decorate(block) {
   image.replaceChildren(block.querySelector('picture'));
   const content = document.createElement('div');
   content.classList.add('hero-content');
-  content.append(header, buildSearch());
+  if (document.querySelector('body.homepage')) {
```
Comment (bstopp) [inline cigaradvisor/blocks/hero/hero.js:29]: Don't check for the search - if there's no search, then that's a hero variation

File: cigaradvisor/blocks/hero/hero.js (hunk 2 above)
Comment (kailasnadh790) [reply cigaradvisor/blocks/hero/hero.js]: fixed

**Review by bstopp — APPROVED** [review]

**Pull request merged**